ng2-components/ng2-activiti-form/src/components/widgets/amount/amount.widget.html

```diff
@@ -14,5 +14,8 @@
                 placeholder="{{field.placeholder}}">
     </md-input-container>
     <error-widget [error]="field.validationSummary" ></error-widget>
+    <div *ngIf="(!field.isValid && !field.validationSummary)">
```

Comment:
All the error widgets have the same container, maybe we just move the div inside error widget, so that it checks "!field.isValid && !field.validationSummary" ?